samples/musiclibrary2/src/main/kotlin/app/cash/tempest2/musiclibrary/MusicDb.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.tempest2.musiclibrary
+
+import app.cash.tempest2.LogicalDb
+import app.cash.tempest2.TableName
+
+interface MusicDb : LogicalDb {
+  @TableName("music_items")
```

Comment:
Breaking change: this annotation is required in tempest2

Reply:
Maybe make it a rule that the field name is music_items?

Reply:
Decided to require this annotation because it is difficult to debug table not found exception.

```
Cannot do operations on a non-existent table (Service: DynamoDb, Status Code: 400, Request ID: 6b8f6f1f-7277-4d39-952c-28eece2f6550, Extended Request ID: null)
software.amazon.awssdk.services.dynamodb.model.ResourceNotFoundException: Cannot do operations on a non-existent table (Service: DynamoDb, Status Code: 400, Request ID: 6b8f6f1f-7277-4d39-952c-28eece2f6550, 
```